kits/arm/ex_impedance_control.py

```diff
@@ -54,11 +51,26 @@
 if enable_logging:
     arm.group.start_log('dir', 'logs', mkdirs=True)
 
-print('Commanded gravity-compensated zero force to the arm.')
-print('  b2 - Toggles an impedance controller on/off:')
-print('          ON  - Apply controller based on current position')
-print('          OFF - Go back to gravity-compensated mode')
-print('  b1 - Exits the demo.')
+# Print Instructions
+instructions = """Commanded gravity-compensated zero force to the arm.
+B1 - Exit Demo
+B2 - Toggles an impedance controller on/off:
+          ON  - Apply controller based on current position
+          OFF - Go back to gravity-compensated mode
+"""
+print(instructions)
+
+m.set_button_label(1, 'Exit')
+m.set_button_label(2, 'Toggle Impedance')
+
+for button in range(3, 9):
+    m.set_button_label(button, '')
+
+for axis in range(1, 9):
+    m.set_axis_label(axis, '')
+  
+m.set_button_mode(2, 1)
```

Comment:
I like that we are resetting the UI here -- this helps ensure a good experience when running different demos.  Two thoughts:
(1) I think there is a "clear" function on the mobile IO class that can help simplify some of this.
(2) This made me realize we don't reset button modes/etc between demos either.  This could be a good thing to reset as well, to get to a fully clean state.  This may be covered in that "reset" function as well.